hibernate-core/src/test/resources/hibernate.properties

```diff
@@ -22,7 +22,7 @@ hibernate.max_fetch_depth 5
 hibernate.cache.region_prefix hibernate.test
 hibernate.cache.region.factory_class org.hibernate.testing.cache.CachingRegionFactory
 
-javax.persistence.validation.mode=NONE
+jakarta.persistence.validation.mode=NONE
 hibernate.service.allow_crawling=false
```

Comment:
This was causing test failures because it set a default for the deprecated setting. Now the property is aligned with `main` branch.